javascript/src/index.ts

```diff
@@ -0,0 +1,54 @@
+import { Point } from "@noble/secp256k1";
```

Comment:
bulk of the logic rests in this file, `utils/` folder at the asme level, and the `test/` folder